packages/component-library/src/components/form/mt-datepicker/mt-datepicker.vue

```diff
@@ -203,63 +209,78 @@ export default defineComponent({
 
   computed: {
     computedValue: {
-      get(): string | string[] {
+      get(): string | string[] | { hours: number; minutes: number } {
```

Comment:
Did you also validate these changes in the Platform directly to be sure that everything works correctly?

Reply:
I'm with @jleifeld here, we should definitely try this fix in the Shopware admin but I didn't get around to testing it yet and we definitely need better documentation on how to test meteor changes in Shopware before they are released...